mypy_extensions.py

```diff
@@ -5,12 +5,32 @@
     from mypy_extensions import TypedDict
 """
 
-from typing import Any, Dict
+from typing import Any, Dict, Final, Literal
 
 import sys
 # _type_check is NOT a part of public typing API, it is used here only to mimic
 # the (convenient) behavior of types provided by typing module.
-from typing import _type_check  # type: ignore
+from typing import _type_check  # type: ignore [attr-defined]
+
+from typing_extensions import NotRequired, TypeGuard, Unpack
+
+
+MYPYC_ATTRS: Final = frozenset([
+    "native_class",
+    "allow_interpreted_subclasses",
+    "serializable",
+])
+
+MypycAttr = Literal[
+    "native_class",
+    "allow_interpreted_subclasses",
+    "serializable",
+]
+
+class MypycAttrs(TypedDict):
+    native_class: NotRequired[bool]
```

Comment:
are there others? I was looking thru the docs and couldn't find any good central resource on this